src/balance/command.ts

```diff
@@ -2,30 +2,65 @@ import { Command } from "commander";
 import Entropy from "@entropyxyz/sdk";
 
 import { EntropyBalance } from "./main";
+import { BalanceInfo } from "./types";
 import { endpointOption, cliWrite, loadEntropy } from "../common/utils-cli";
 import { findAccountByAddressOrName, getTokenDetails, nanoBitsToBits, round } from "../common/utils";
 import * as config from "../config";
+import { EntropyConfigAccount } from "src/config/types";
 
 export function entropyBalanceCommand () {
   const balanceCommand = new Command('balance')
+  // account can no longer be a required argument if a user wishes to
+  // view the balances of all accounts
   balanceCommand
     .description('Command to retrieive the balance of an account on the Entropy Network')
-    .argument('<account>', [
+    .argument('[account]', [
       'The address an account address whose balance you want to query.',
       'Can also be the human-readable name of one of your accounts'
     ].join(' '))
+    .option('-a, --all', 'Get balances for all admin accounts in the config')
     .addOption(endpointOption())
     .action(async (account, opts) => {
-      const entropy: Entropy = await loadEntropy(account, opts.endpoint)
+      const { accounts } = await config.get()
+
+      let entropy: Entropy
+      if (!account && opts.all) {
+        const tempAddress = accounts[0].address
+        entropy = await loadEntropy(tempAddress, opts.endpoint)
+      } else if (account) {
+        entropy = await loadEntropy(account, opts.endpoint)
+      } else {
+        balanceCommand.help()
+        return process.exit(0)
+      }
```

Comment:
:fire: this still has a gap: 
```bash
entropy balance naynay -a
```
`!account && opts.all` => `false`
`account` => `true`  but I specified `-a` as well... should not be allowed

So perhaps
```suggestion
      if (!account && opts.all) {
        const tempAddress = accounts[0].address
        entropy = await loadEntropy(tempAddress, opts.endpoint)
      } else if (account && !opts.all) {
        entropy = await loadEntropy(account, opts.endpoint)
      } else {
        balanceCommand.help()
        return process.exit(0)
      }
```